benefits/eligibility/views.py

```diff
@@ -146,66 +146,67 @@ def confirm(request):
         eligibility = session.eligibility(request)
         return verified(request, [eligibility.name])
 
+    agency = session.agency(request)
     verifier = session.verifier(request)
+    types_to_verify = verify.typenames_to_verify(agency, verifier)
 
     # GET for OAuth verification
     if request.method == "GET" and verifier.uses_auth_verification:
-        analytics.started_eligibility(request)
+        analytics.started_eligibility(request, types_to_verify)
 
-        verified_types = verify.eligibility_from_oauth(verifier, session.oauth_claim(request), session.agency(request))
+        verified_types = verify.eligibility_from_oauth(verifier, session.oauth_claim(request), agency)
         if verified_types:
             return verified(request, verified_types)
         else:
             return unverified(request)
 
     # GET/POST for Eligibility API verification
-    else:
```

Comment:
Dropped this outer `else` here since it wasn't necessary and made the code less readable. By this point, the user is known to be unverified, and we are not using "auth verification".

The only further distinction is `GET` vs. `POST`